data/strings/types_strings.txt

```diff
@@ -14712,7 +14715,7 @@
     ca = Torre de comunicacions
     cs = Komunikační věž
     da = Kommunikationstårn
-    de = Kommunikationsturm
+    de = Mobilfunkmast
```

Comment:
[tower:type=communication](https://wiki.openstreetmap.org/wiki/DE:Tag:tower:type%3Dcommunication) doesn't necessarily mean its a cell tower, it could as well be a radio or television tower. Therefore I would keep "Kommunikationsturm".

Reply:
or change it to Funkmast/Funkturm as it is called in the Wiki